Tools/autotest/common.py

```diff
@@ -1,83 +1,99 @@
-import util, pexpect, time, math
+import math
+import time
+
 from pymavlink import mavwp
 
+from pysim import util
+
 # a list of pexpect objects to read while waiting for
 # messages. This keeps the output to stdout flowing
 expect_list = []
 
+
 def expect_list_clear():
-    '''clear the expect list'''
+    """clear the expect list."""
     global expect_list
     for p in expect_list[:]:
         expect_list.remove(p)
 
-def expect_list_extend(list):
-    '''extend the expect list'''
+
+def expect_list_extend(list_to_add):
+    """Extend the expect list."""
     global expect_list
-    expect_list.extend(list)
+    expect_list.extend(list_to_add)
+
 
 def idle_hook(mav):
-    '''called when waiting for a mavlink message'''
+    """Called when waiting for a mavlink message."""
     global expect_list
     for p in expect_list:
         util.pexpect_drain(p)
 
+
 def message_hook(mav, msg):
-    '''called as each mavlink msg is received'''
+    """Called as each mavlink msg is received."""
     idle_hook(mav)
 
+
 def expect_callback(e):
-    '''called when waiting for a expect pattern'''
+    """Called when waiting for a expect pattern."""
     global expect_list
     for p in expect_list:
         if p == e:
             continue
         util.pexpect_drain(p)
 
+
 def get_distance(loc1, loc2):
-    '''get ground distance between two locations'''
-    dlat 		= loc2.lat - loc1.lat
-    dlong		= loc2.lng - loc1.lng
+    """Get ground distance between two locations."""
+    dlat        = loc2.lat - loc1.lat
+    dlong       = loc2.lng - loc1.lng
```

Comment:
A single space after dlong and align above? I don't see no reason for all this space.